src/voronoi.rs

```diff
@@ -9,6 +9,10 @@ use std::convert::TryInto;
 pub fn compute_voronoi<T: Borrow<Point<f64>>>(
     points: &[T],
     envelope: Option<&GGeometry>,
+    /// From the underlying c library
+    /// "snapping tolerance to use for improved robustness. A tolerance of 0.0 specifies that no snapping
+    /// will take place. This argument can be finicky and is known to cause the algorithm to fail in
+    /// several cases. If you're using tolerance and getting a failure, try setting it to 0.0."
```

Comment:
Doc comments are not supported on function arguments, so you'll need to put it on the function. Also, we could use "quote" syntax instead:

```
    /// From the underlying C library:
    /// > snapping tolerance to use for improved robustness. A tolerance of 0.0 specifies that no snapping
    /// > will take place. This argument can be finicky and is known to cause the algorithm to fail in
    /// > several cases. If you're using tolerance and getting a failure, try setting it to 0.0.
```

Reply:
I can do that, how does the new version look?